<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_1">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1300</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid white;</string>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="game1Button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>35</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Asap Condensed</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:focus { outline-color:white; 
border:1px solid #ffffff;}
QPushButton{
background:transparent;
border:none;
color:white;
}
</string>
     </property>
     <property name="text">
      <string>GAME 1</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="game2Button">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>35</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Asap Condensed</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:focus { outline-color:white; 
border:1px solid #ffffff;}
QPushButton{
background:transparent;
border:none;
color:white;
}
</string>
     </property>
     <property name="text">
      <string>GAME 2</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="game3Button">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>35</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Asap Condensed</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:focus { outline-color:white; 
border:1px solid #ffffff;}
QPushButton{
background:transparent;
border:none;
color:white;
}
</string>
     </property>
     <property name="text">
      <string>GAME 3</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="emptyButton">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>35</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Asap Condensed</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:focus { outline-color:white; 
border:1px solid #ffffff;}
QPushButton{
background:transparent;
border:none;
color:white;
}
</string>
     </property>
     <property name="text">
      <string>EMPTY</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="profileButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>35</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Asap Condensed</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:focus { outline-color:white; 
border:1px solid #ffffff;}
QPushButton{
background:transparent;
border:none;
color:white;
}
</string>
     </property>
     <property name="text">
      <string>PROFILE</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="signOutButton">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>35</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Asap Condensed</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:focus { outline-color:white; 
border:1px solid #ffffff;}
QPushButton{
background:transparent;
border:none;
color:white;
}
</string>
     </property>
     <property name="text">
      <string>SIGN OUT</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1280</width>
     <height>640</height>
    </rect>
   </property>
   <widget class="QWidget" name="page_3"/>
   <widget class="QWidget" name="page_4"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
